.github/workflows/main.yml

```diff
@@ -102,3 +107,8 @@ jobs:
           state: 'success'
           sha: ${{github.event.pull_request.head.sha || github.sha}}
           target_url: https://snapshots.nextjournal.com/clerk/build/${{ github.sha }}
+
+      - name: Run Playwright tests against static assets
+        run: |
+          bb test:static-app ${{ github.sha }}
```

Comment:
Can we make this run as a separate task, producing a separate GitHub status badge? I'd like to be able to differentiate between a build failure of the static app vs a failure of the browser tests.

Reply:
Done